File: ios/MyTatva/ReactNativeCommon/RNShare.m
//
//  RNShare.m
//  MyTatva
//
//  Created by Macbook Pro on 13/09/23.
//

#import <Foundation/Foundation.h>
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>
#import <React/RCTLog.h>
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>
#import <React/RCTLog.h>
#import "React/RCTEventEmitter.h"
#import <React/RCTBridge.h>
#import <React/RCTEventDispatcher.h>
#import <React/RCTRootView.h>
#import <React/RCTConvert.h>
#import <React/RCTBundleURLProvider.h>

@interface RCT_EXTERN_MODULE(RNShare, NSObject)

RCT_EXTERN_METHOD(getToken: (RCTPromiseResolveBlock)resolve rejecter: (RCTPromiseRejectBlock)reject)

+ (BOOL) requiresMainQueueSetup {
  return YES;
}

@end

File: ios/MyTatva/ReactNativeCommon/RNShare.swift
//
//  CustomMethods.swift
//  MyTatva
//
//  Created by Macbook Pro on 13/09/23.
//

import Foundation

@objc(RNShare)
class RNShare : NSObject {
    @objc
    func constantsToExport() -> [AnyHashable : Any]! {
        return ["token": UserModel.accessToken ?? "","hide_incident_survey":hide_incident_surveyMain]
    }
    
    @objc static func requiresMainQueueSetup() -> Bool {
        return true
    }
}
